Jenkinsfile

```diff
@@ -57,7 +57,7 @@ def cmake_build(compiler, flags, prefixpath="/opt/rocm"){
     }
 }
 
-def buildJob(compiler, flags, image, prefixpath="/opt/rocm", cmd = ""){
+def buildJob(compiler, flags, env4make, image, prefixpath="/opt/rocm", cmd = ""){
```

Comment:
@atamazov How can I set separate set of environment variables for different test target such as test_conv_for_implicit_gemm, test_conv2d?

As I undertand, the env4make only allows to set separate env variables for diferrent jobs, not different env for different test target within a job.

Reply:
This is what you can do: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/240/files#r431515058

Reply:
Perhaps it is time to move away from bulk `make check` and atomize our tests so that there isn’t so much redundancy.

Reply:
Perhaps it is time to move away from bulk `make check` and atomize our tests so that there isn’t so much redundancy.

Reply:
@TejashShah 
> How can I set separate set of environment variables for different test target such as test_conv_for_implicit_gemm, test_conv2d?

Please see https://cmake.org/cmake/help/v3.10/manual/cmake-properties.7.html#properties-on-tests, there is `ENVIRONMENT` property. How to set it is another question ;)